examples/tests/efficiency.c

```diff
@@ -0,0 +1,104 @@
+
+#include <igraph.h>
+
+#include "test_utilities.inc"
+
+/* Number of edges in the test graph.
+   This value, 'graph', and 'weights_array' must be consistent. */
+#define N_EDGES 24
+
+int main() {
+    igraph_t graph;
+    const igraph_real_t weights_array[] = {4, 4, 4, 3, 1, 5, 1, 2, 4, 5, 3, 5, 5, 4, 1, 1, 5, 4, 1, 1, 2, 1, 3, 5};
```

Comment:
add `N_EDGES` here between the brackets and then we can be sure that `N_EDGES` is consistent with the length of the array

Reply:
Done.